azure-pipelines.yml

```diff
@@ -467,7 +467,7 @@ stages:
           CSharp:
             testClasses: 'com.sonar.it.csharp.Tests'
           Others:
```

Comment:
Not really a comment for this PR. Why don't we have a matrix of CSharp, VBNet and Shared?

Reply:
Because `VBNET`+`Shared` are faster than `CSharp`. So it's fine to split them like this for now. This parallelization was done quite recently.